<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HexKeyboard</class>
 <widget class="QWidget" name="HexKeyboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>30</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton {
	font-size: 30px;
	background-color: #ffffff;
	color: #0c2731;
	border: 2px solid black;
	border-radius: 15px;
}

QToolButton:pressed {
	background-color: #f0f0f0;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QToolButton" name="characterA">
     <property name="maximumSize">
      <size>
       <width>92</width>
       <height>82</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>A</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextOnly</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="characterE">
     <property name="maximumSize">
      <size>
       <width>92</width>
       <height>82</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>E</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextOnly</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QToolButton" name="characterB">
     <property name="maximumSize">
      <size>
       <width>92</width>
       <height>82</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>B</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextOnly</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="characterF">
     <property name="maximumSize">
      <size>
       <width>92</width>
       <height>82</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>F</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextOnly</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QToolButton" name="characterC">
     <property name="maximumSize">
      <size>
       <width>92</width>
       <height>82</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>C</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextOnly</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="characterX">
     <property name="maximumSize">
      <size>
       <width>92</width>
       <height>82</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>x</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextOnly</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QToolButton" name="characterD">
     <property name="maximumSize">
      <size>
       <width>92</width>
       <height>82</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>D</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextOnly</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
